### PR TITLE
Feature/update cli to match latest drop contracts

### PR DESCRIPTION
**Pull request by alexanderattar**
## Describe Changes

- Update the contract name and references
- Update the ABIs
- Update the bytecode
- Refactor the initcode functions to me more modular while maintaining more simplicity
  - The previous attempt added too much boilerplate functions for generating tuples that actually made the code a lot harder to debug. I believe this second attempt keeps the code more organized without introducing unnecessary complexity

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] I have checked eslint

### CODE REVIEW

File: src/commands/create/contract.ts [hunk 1]
```diff
@@ -558,8 +560,8 @@ export default class Contract extends Command {
     const receipt: TransactionReceipt | null = await this.networkMonitor.executeTransaction({
       network: targetNetwork,
       // NOTE: gas can be overriden by here
-      // gasPrice: ethers.BigNumber.from(100000000000), // 100 gwei
-      // gasLimit: ethers.BigNumber.from(7000000), // 7 million
+      gasPrice: ethers.BigNumber.from(100_000_000_000), // 100 gwei
```
Comment (alexanderattar) [inline src/commands/create/contract.ts]: temporarily overriding here to force the tx to go through

File: src/commands/create/contract.ts [hunk 2]
```diff
@@ -209,8 +214,8 @@
           case BytecodeType.SampleERC721:
             contractType = 'HolographERC721'
             break
-          case BytecodeType.HolographDropsEditionsV1:
-            contractType = 'HolographDropsEditionsV1'
+          case BytecodeType.HolographDropERC721:
+            contractType = 'HolographDropERC721'
```
Comment (sogoiii) [inline src/commands/create/contract.ts]: would be it worth it to put `contractType = BytecodeType.HolographDropERC721` instead?

File: src/commands/create/contract.ts [hunk 3]
```diff
@@ -122,7 +127,7 @@
     let chainId: string
     let salt: string
     let bytecodeType: BytecodeType
-    const contractTypes: string[] = ['HolographERC20', 'HolographERC721', 'HolographDropsEditionsV1']
+    const contractTypes: string[] = ['HolographERC20', 'HolographERC721', 'HolographDropERC721']
```
Comment (sogoiii) [inline src/commands/create/contract.ts]: should we reference the values in `BytecodeType.HolographDropERC721` and such?

File: src/commands/create/contract.ts [hunk 4]
```diff
@@ -340,7 +345,7 @@
 
             break
 
-          case 'HolographDropsEditionsV1': {
+          case 'HolographDropERC721': {
```
Comment (sogoiii) [inline src/commands/create/contract.ts]: should we set this as `BytecodeType.HolographDropERC72`? and other cases as well.

File: src/commands/create/contract.ts [hunk 5]
```diff
@@ -445,32 +449,38 @@
             const account = userWallet.connect(provider)
 
             // Deploy a metadata renderer contract
-            const renderAbi = JSON.parse(fs.readFileSync(`./src/abi/develop/EditionMetadataRenderer.json`).toString())
-            const rendererFactory = new ContractFactory(renderAbi, EditionMetadataRenderer.bytecode, account)
+            // TODO: this needs to be removed in the future and a reference to the deployed EditionsMetadataRendererProxy needs to be made here
+            const renderAbi = JSON.parse(fs.readFileSync(`./src/abi/develop/EditionsMetadataRenderer.json`).toString())
+            const rendererFactory = new ContractFactory(renderAbi, EditionsMetadataRenderer.bytecode, account)
```
Comment (sogoiii) [inline src/commands/create/contract.ts]: So we have the method getABIs that is a large map of contracts. Shold we add that there? This hardcodes develop as well. Which i guess ok for now ,but we should make sure to update this before promoting to testnet.

File: src/commands/create/nft.ts [hunk 6]
```diff
@@ -126,8 +126,8 @@ export default class NFT extends Command {
       case 'CxipERC721':
         abiPath = path.join(__dirname, `../../abi/${environment}/CxipERC721.json`)
         break
-      case 'HolographDropsEditionsV1':
-        abiPath = path.join(__dirname, `../../abi/${environment}/HolographDropsEditionsV1.json`)
+      case 'HolographDropERC721':
+        abiPath = path.join(__dirname, `../../abi/${environment}/HolographDropERC721.json`)
```
Comment (sogoiii) [inline src/commands/create/nft.ts]: Should we grab this from the `getABIs` function?

**Review by sogoiii — CHANGES_REQUESTED** [review]
I think if we should convert some of those hardcoded string references in to variables.

**Review by sogoiii — APPROVED** [review]
👍

**Pull request merged**